tests/input/example-class.php

```diff
@@ -61,6 +61,9 @@ public function isBaz() : bool
         return $this->baz;
     }
 
+    /**
+     * @throws \InvalidArgumentException if this example cannot baz.
```

Comment:
Shouldn't be this instead (otherwise we're mixing sniffs fixes)?

```diff
- @throws \InvalidArgumentException if this example cannot baz.
+ @throws InvalidArgumentException If this example cannot baz.
```

Reply:
Hmmm yes but it should be `@throws InvalidArgumentException if this example cannot baz.` then so that we can test non-capital won't trigger an error.

Reply:
@deeky666 this file represents the input of the test, so IMO if we want to standardise things I'd expect to have a sniff that fixes things automatically (so the `input` would contain things in capital which would be fixed in the `output`).

Reply:
@lcobucci I get that but there isn't really anything to fix here because we do not explicitly disallow capital letters. How can we test then, that this sniff is excluded now?

Reply:
@deeky666 so the question is: do we want to have it standardised or not? If yes, we should have a sniff and create the correct expectations.

We should have had tests to cover the current behaviour, though.